python/sgtk/__init__.py

```diff
@@ -17,7 +17,7 @@
 
 # now go through and duplicate all entries in sys.modules 
 import sys
-for x in sys.modules.keys():
+for x in list(sys.modules.keys()):
```

Comment:
Maybe add a comment here explaining why you're creating a list out of those keys. I think it has to do with the fact that we're updating the modules dict, right?

Reply:
✅ added.